spark-jobs/src/test/scala/za/co/absa/enceladus/standardization/StandardizationParquetSuite.scala

```diff
@@ -436,6 +437,32 @@ class StandardizationParquetSuite extends FixtureAnyFunSuite with SparkTestBase
     assert(actual == expected)
   }
 
+
+  test("Timestamp with timezone in metadata are shifted") {tmpFileName =>
+    val args = (s"--dataset-name $datasetName --dataset-version $datasetVersion --report-date 2019-07-23" +
+      " --report-version 1 --menas-auth-keytab src/test/resources/user.keytab.example " +
+      "--raw-format parquet").split(" ")
+
+    val expected =
+      """+-------------------+------+
+        ||ts                 |errCol|
+        |+-------------------+------+
+        ||1969-12-31 23:00:00|[]    |
+        ||1969-12-31 22:00:00|[]    |
```

Comment:
I'm sorry, maybe I am missing the point here, but I will try:
It seems counter-intuitive that dataset with `(UTC midnight and CET midnight)` can end up as `(11pm CET and 10pm CET)`. 

If this case is truly correct, a bit of explanatory comment could help.

Reply:
I see, I will add a comment. But is correct. Let me try to explain:
The timestamps are not `(UTC midnight and CET midnight)`, that's just their source. They are timestamp without timezone, one saying 12:00:00AM the other 23:00:00PM. And the _metadata_ suggests the timestamp is in CET for the conversion. Then displayed in UTC...

Reply:
Oh, I see. Yes, commenting in this fashion would help, then.